packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -6,10 +6,9 @@ const fastcsv = require('fast-csv');
 const inquirer = require('inquirer');
 const debug = require('debug')('export-to-csv');
 const checkboxPlus = require('inquirer-checkbox-plus-prompt');
-
 const config = require('./config.js');
-const { cliux, configHandler } = require('@contentstack/cli-utilities');
-
+const { cliux, configHandler, HttpClient } = require('@contentstack/cli-utilities');
+const _ = require('lodash');
```

Comment:
@cs-raj Instead of loading the full library, we can retrieve a specific method from Lodash. like const find = require('lodash/find');

Reply:
Imported clone deep only